### ISSUE TITLE
Clarify install 

### ISSUE THREAD

**Issue by pedrocunha** [miketheman/heroku-buildpack-datadog#12]
Hi,

Thanks for the buildpack it works wonders. 

I think it would be helpful to add in the README you must do:

```
heroku buildpacks:set https://github.com/ddollar/heroku-buildpack-multi.git
```

Thanks

**Comment by miketheman** [comment]
Hi @pedrocunha !

I was reading the Heroku docs and it seems like this stop is no longer needed - and is simpler overall:
https://devcenter.heroku.com/articles/buildpacks
https://devcenter.heroku.com/articles/using-multiple-buildpacks-for-an-app

So the `multi` buildpack is no longer required, and the docs should definitely be updated to do the right installation procedure.

**Comment by pedrocunha** [comment]
Cheers Mike, appreciated.

**Comment by miketheman** [comment]
Closed via #13.
